project/Dependencies.scala

```diff
@@ -55,8 +55,9 @@ object Dependencies {
   val caliper       = ("com.google.code.caliper" % "caliper" % "1.0-SNAPSHOT"
     from "http://plastic-idolatry.com/jars/caliper-1.0-SNAPSHOT.jar")
 
-  val nscalaTime    = "com.github.nscala-time" %% "nscala-time" % "1.6.0"
+  val nscalaTime    = "com.github.nscala-time" %% "nscala-time" % "2.6.0"
 
+//  val awsSdkS3      = "com.amazonaws" % "aws-java-sdk-s3" % "1.10.11"
   val awsSdkS3      = "com.amazonaws" % "aws-java-sdk-s3" % "1.9.34"
```

Comment:
If by "change my mind" you mean agonizingly run through versions of this dependency that would both recognize `eu-central-1` as a region and not throw signature or other crazy errors, then yes :)